index.html

```diff
@@ -7,12 +7,147 @@
+
+            <div class="program-card">
+                <img src="./images/program_icon_01.svg" alt="program One" class="program-icon">
+                <h3 class="program-name">Lecture</h3>
+                <p class="program-desc">Listen to the speakers from various countries about the messages of sharing and opening.</p>
+            </div>
+
+            <div class="program-card">
+                <img src="./images/program_icon_02.svg" alt="program One" class="program-icon">
+                <h3 class="program-name">CC Exhibition</h3>
+                <p class="program-desc">Appreciate various creations applying CC license of artists, organized from Art Center Nabi</p>
+            </div>
+
+            <div class="program-card">
+                <img src="./images/program_icon_03.svg" alt="program One" class="program-icon">
+                <h3 class="program-name">Forum</h3>
+                <p class="program-desc">Have the time to share your thoughts and opinions with experts for each topic.</p>
+            </div>
+
+            <div class="program-card">
+                <img src="./images/program_icon_04.svg" alt="program One" class="program-icon">
+                <h3 class="program-name">WorkShop</h3>
+                <p class="program-desc">Try creating your own work using open source license rather than just watching at it.
+                </p>
+            </div>
+
+            <div class="program-card">
+                <img src="./images/program_icon_05.svg" alt="program One" class="program-icon">
+                <h3 class="program-name">CC Ignite</h3>
+                <p class="program-desc">get opportunities to network with CC affiliates around the world, also after the summit.</p>
+            </div>
+
+        </div>
+            <a href="about.html" class="join-btn">Join JA Kenya 2023</a>
+            <a href="#" class="join-btn2">SEE THE WHOLE PROGRAM</a>
+        </div>
+    </section>
+
+    <section class="speakers">
+        <div class="speaker-wrapper">
+            <h2>Featured Speakers</h2>
+            <hr />
+
+            <div class="all-speakers">
+           <!-- Dynamically created in html -->
+        </div>
+
+            <button href="#" onclick="loadData()" class="see-more">MORE<span></span><i class="uil uil-angle-down"></i></button>
+        </div>
+
+        <section class="partner">
+            <h2>Partner</h2>
+            <hr />
+            <div class="partner-logos">
+                <img src="./images/airbnb.svg" alt="">
+                <img src="./images/daumkakaologo.svg" alt="">
+                <img src="./images/googlekorea.svg" alt="">
+                <img src="./images/mozilla.svg" alt="">
+                <img src="./images/naver.svg" alt="">
+            </div>
+        </section>
     
+        <footer>
+            <img src="./images/cck-logo1.svg" alt="">
+            <p>2015 Creative Commons Korea. Some Rights Reserved.</p>
+        </footer>
```

Comment:
- [ ] As suggested by the previous reviewer, please make sure to personalize the content of your webpage ✨ You have done a great job in personalizing other sections, please personalize this as well 😄